docs/src/content/docs/es/index.mdx

```diff
@@ -45,6 +47,113 @@ import AboutAstro from '~/components/about-astro.astro';
 	</Card>
 </CardGrid>
 
+<TestimonialGrid title="Qué es lo que dice la gente">
```

Comment:
"gente" is a touch informal for my taste but I couldn't decide on a good alternative. Here are some options:
- Lo que dicen las personas
- Lo que expresan las personas

I am also open to other alternatives

Reply:
What do you think about "Qué es lo que opinan lo demás"?

Reply:
Feel free to interpret it liberally! Excuse my bad Spanish, but just to show how free we can be, it could also be flipped around to something like “Opiniones que hemos oído” for example if that worked better in translation. No need to be literal on a heading like this that is kind of punchy and more “marketing” in context.

Reply:
Marketing is hard 😅

Reply:
Yeah, it can be 😁 which suggestion would make more sense:
- Qué es lo que opinan lo demás
- Opiniones que hemos oído
- Lo que estamos oyendo

Reply:
I like the style of "Lo que estamos oyendo", I tweak it a bit to follow the translation. "Lo que nuestros usuarios opinan" What do you think? @dreyfus92